.github/workflows/build.yml

```diff
@@ -12,6 +12,10 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
+      - "docs/**"
+      - "logo/**"
+      - "*.md"
```

Comment:
This might be tricky since we are going to be having md files within the sdk directory.

Let's remove this filter for now

Reply:
This only filters the root level .md files, so it should be fine. The general query syntax should be **.md.